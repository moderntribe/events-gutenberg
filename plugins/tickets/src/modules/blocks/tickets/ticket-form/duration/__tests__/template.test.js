/**
 * External dependencies
 */
import React from 'react';

/**
 * Internal dependencies
 */
import TicketDurationPicker from './../template';

describe( 'Ticket Duration picker and label', () => {
	test( 'default properties', () => {
		const component = renderer.create( <TicketDurationPicker /> );
		expect( component.toJSON() ).toMatchSnapshot();
	} );
} );
