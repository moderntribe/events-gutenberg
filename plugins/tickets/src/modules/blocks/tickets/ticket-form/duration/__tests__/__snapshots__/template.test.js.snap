// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Ticket Duration picker and label default properties 1`] = `
<div
  className="tribe-editor__tickets-form__row"
>
  <div
    className="tribe-editor__label-with-tooltip tribe-editor__tickets-form__label"
  >
    <span
      className="tribe-editor__label-with-tooltip__label"
    >
      Sale Duration
    </span>
    <span>
      Tooltip
    </span>
  </div>
  <div
    className="tribe-editor__tickets-form__input-group"
  >
    <div
      className="tribe-editor__date-time-range-picker tribe-editor__ticket-duration__duration-picker"
    >
      <div
        className="tribe-editor__date-time-range-picker__start"
      >
        <div
          className="DayPickerInput"
        >
          <input
            onBlur={[Function]}
            onChange={[Function]}
            onClick={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            placeholder="YYYY-M-D"
            value="September 3, 2018"
          />
        </div>
        <span
          className="tribe-editor__date-time-range-picker__separator tribe-editor__date-time-range-picker__separator--date-time"
        >
          at
        </span>
        <div
          className="tribe-editor__timepicker"
        >
          <span>
            Dropdown
          </span>
        </div>
      </div>
      <div
        className="tribe-editor__date-time-range-picker__end"
      >
        <span
          className="tribe-editor__date-time-range-picker__separator tribe-editor__date-time-range-picker__separator--time-range"
        >
          to
        </span>
        <div
          className="DayPickerInput"
        >
          <input
            onBlur={[Function]}
            onChange={[Function]}
            onClick={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            placeholder="YYYY-M-D"
            value="September 6, 2018"
          />
        </div>
        <span
          className="tribe-editor__date-time-range-picker__separator tribe-editor__date-time-range-picker__separator--date-time"
        >
          at
        </span>
        <div
          className="tribe-editor__timepicker"
        >
          <span>
            Dropdown
          </span>
        </div>
      </div>
    </div>
  </div>
</div>
`;
